gui/src/daemon/model.rs

```diff
@@ -122,6 +127,13 @@ impl SpendTx {
     pub fn is_self_send(&self) -> bool {
         !self.coins.is_empty() && self.spend_amount == Amount::from_sat(0)
     }
+
+    /// Feerate obtained if all transaction inputs have the maximum satisfaction size.
+    pub fn min_feerate_vb(&self) -> u64 {
+        let max_tx_vbytes =
+            self.psbt.unsigned_tx.vsize() + (self.max_sat_vbytes * self.psbt.inputs.len());
+        self.fee_amount.to_sat() / max_tx_vbytes as u64
+    }
```

Comment:
Could you add a comment about how this assumes all inputs are internal for this computation? I think it's worth having for the future.

(Also that's an ok assumption in the context of this estimation.)

Reply:
I've added a comment in https://github.com/wizardsardine/liana/pull/725/commits/846431ed53efd2b29aefabe84ed56d054c0d4b45.